modules/core/include/opencv2/core/mat.hpp

```diff
@@ -2134,6 +2147,7 @@ class CV_EXPORTS Mat
     int dims;
     //! the number of rows and columns or (-1, -1) when the matrix has more than 2 dimensions
     int rows, cols;
+    int dummy = 153;
```

Comment:
?

Reply:
I will remove it in one of the subsequent commits in a different pull request;
Let it stay here for a few weeks/months to catch the situation when Mat.size.p (when it points to rows or cols) is accessed outside of those 2 fields